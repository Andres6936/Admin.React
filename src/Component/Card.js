import React from 'react';

export class Card extends React.Component {
    render() {
        return <div className={"card rounded shadow-sm py-2 mb-4"}>
            {this.props.children}
        </div>
    }
}
