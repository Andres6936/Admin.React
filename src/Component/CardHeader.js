import React from 'react';

export class CardHeader extends React.Component {
    render() {
        return (
            <div className={"card-header text-primary bg-white fw-bold"}>
                {this.props.title}
                {this.props.children}
            </div>
        );
    }
}
