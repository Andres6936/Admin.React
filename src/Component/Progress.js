import React from 'react';

export class Progress extends React.Component {
    render() {
        return (
            <div className="progress mb-3">
                <div className={"progress-bar " + this.props.colorBar} role="progressbar"
                     style={{width: this.props.percentage + '%'}}
                     aria-valuenow={this.props.percentage} aria-valuemin="0" aria-valuemax="100">

                </div>
            </div>
        );
    }
}
