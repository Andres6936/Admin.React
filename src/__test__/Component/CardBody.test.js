import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from "enzyme-to-json";
import {CardBody} from "../../Component/CardBody";


Enzyme.configure({adapter: new Adapter()})

describe('<CardBody/>', function () {
    it("Snapshot for default Row component", () => {
        const wrapper = shallow(<CardBody/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })

    it("Render for default Row component", () => {
        const wrapper = shallow(<CardBody/>);
        expect(wrapper.contains(<div className={"card-body"}/>)).toEqual(true);
    })
})
