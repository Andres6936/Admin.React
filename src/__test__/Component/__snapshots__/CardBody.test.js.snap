// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardBody/> Snapshot for default Row component 1`] = `
<div
  className="card-body"
/>
`;
